### ISSUE TITLE
MSFT-83[Member] - The contrast color is low for Unregistered Button

### ISSUE THREAD

**Issue by veshu** [veshu/testjiraimport#92]
**Steps to Reproduce**
1. Go to https://www.topcoder.com/challenges/XXXXXXX
2. Inspect Element
3. Observe the color of Unregistered Button.

**Expected Result**
Expected contrast ratio of  4:5:1.

**Actual Result**
Element has insufficient color contrast Ration 2.5:1 (foreground color: #FFFFFF, background color: ###ff5b52, Contrast Ratio : 3.06:1)

**Device**: Desktop/Labtop
**Operating System**: macOS
**Browser**: Chrome
**WCAG Category**: 1. Perceivable
**WCAG Level**: AA
**WCAG Success Criterion**: 1.4.3 Contrast (minimum)

**Attachment**
https://topcodermsft-my.sharepoint.com/personal/pd-topcoder_topcodermsft_onmicrosoft_com/Documents/Forms/All.aspx?cid=ae14bf9a%2D590b%2D4bb4%2Da40a%2D4c6822c880a4&FolderCTID=0x0120005C598A51049FC14CBA882E1AEE168F51&id=%2Fpersonal%2Fpd%2Dtopcoder%5Ftopcodermsft%5Fonmicrosoft%5Fcom%2FDocuments%2FMSFT%2DTeams%2DQA%2FJune%202019%2F29%20Jun%2FTest%20Results%2FBug%20Videos%20%26%20Log%20Files%2FCreeya%2FBug%2083
